app/views/topics/show.html.erb

```diff
@@ -10,8 +10,9 @@
       <%-if (i+1) < @breadcrumbs.length%>
         itemref="breadcrumb-<%=(i+1)%>"
       <%-end%>>
-      <a href="<%=c[:url]%>" itemprop="url">
-        <span itemprop="title"><%=c[:name]%></span>
+      <a href="<%=c[:url]%>" itemprop="url" class='badge-wrapper bullet'>
+        <span class="badge-category-bg" style='background-color: #CEA9A9;'> </span>
```

Comment:
We should avoid inline styles wherever possible. Is there a reason this couldn't be done in a SCSS file?